internal/interfaces/grpc/handler/account.go

```diff
@@ -21,20 +21,19 @@ func NewAccountHandler(appSvc *application.AccountService) pb.AccountServiceServ
 func (a *account) CreateAccountBIP44(
 	ctx context.Context, req *pb.CreateAccountBIP44Request,
 ) (*pb.CreateAccountBIP44Response, error) {
-	name, err := parseAccountName(req.GetName())
-	if err != nil {
-		return nil, status.Error(codes.InvalidArgument, err.Error())
-	}
-
-	accountInfo, err := a.appSvc.CreateAccountBIP44(ctx, name)
+	accountInfo, err := a.appSvc.CreateAccountBIP44(ctx, req.GetLabel())
 	if err != nil {
 		return nil, err
 	}
+
 	return &pb.CreateAccountBIP44Response{
-		AccountName:    accountInfo.Key.Name,
-		AccountIndex:   accountInfo.Key.Index,
-		Xpub:           accountInfo.Xpub,
-		DerivationPath: accountInfo.DerivationPath,
+		AccountInfo: &pb.AccountInfo{
+			Label:          req.GetLabel(),
```

Comment:
let's source the label from account's info:
```suggestion
			Label:          accountInfo.Label,
```